src/main/java/io/vlingo/schemata/Bootstrap.java

```diff
@@ -68,7 +68,7 @@ public Bootstrap(final String runtimeType) throws Exception {
     server = Server.startWith(world.stage(),
       allResources,
       SCHEMATA_PORT,
-      Configuration.Sizing.define().withDispatcherPoolSize(2).withMaxMessageSize(16777215),
+      Configuration.Sizing.define().withDispatcherPoolSize(2).withMaxBufferPoolSize(100).withMaxMessageSize(1048576),
```

Comment:
@wwerner Does this seem a bit big still? My guess is that requests and responses normally require `4096` or less bytes to transfer data. It's a shame to use 200 MB of RAM when 1 MB would do for 99% of the use cases. This is why I suggested supporting custom buffer sizes for certain use cases. For example, what if we had two vlingo-http `Server`s on two different ports, and the really big requests/responses must go through the second `Server` port? That second `Server` would be configured with a smaller pool of larger buffers:

`poolSize = 20`
`messageSize = 1048576`

Reply:
Ah, _now_ I get you. This is not a common thing to do, but I guess we can make it work by sending the correct [CORS](https://developer.mozilla.org/de/docs/Web/HTTP/CORS) headers from the server. Otherwise, the browser running the client would forbid it as a cross site request. I'll check tomorrow whether just setting the headers works, otherwise we'll need to implement the complete CORS lifecyle including prefilght requests and so on. In that case, `vlingo-http` would be the correct place for an implementation, I think.

No dynamic buffers to the rescue?

Reply:
@wwerner I think there may be a way to support custom response buffer sizes. We don't need custom request buffers because the server will handle partial reads ss segments when the smaller buffer is full.

Reply:
Is it expected that we need to start a separate `world` for each server? If I try to run two servers within the same `world` (in a single or separate stages), requests hang with no indication of an error.

To reproduce, see https://github.com/vlingo/vlingo-schemata/blob/sandbox/multiple-servers/src/main/java/io/vlingo/schemata/BootstrapMultiple.java

Reply:
After https://github.com/vlingo/vlingo-http/commit/5b9ea57d644e902e4687bd43be22b4145ea9f1c4, trying to send messages  exceeding `maxMessageSize` do no longer lead to overflows, as in this case, the buffer is dynamically grown. No two servers required.